decidim-core/app/models/decidim/amendment.rb

```diff
@@ -16,6 +16,16 @@ def rejected?
       state == "rejected"
     end
 
+    def promoted?
+      return unless rejected?
+      promoted_to_proposal.present?
+    end
+
+    def promoted_to_proposal
```

Comment:
I don't understand the naming of this method :/

Reply:
Any suggestion?

Reply:
Refactored the code: [5eeabdd](https://github.com/decidim/decidim/pull/5178/commits/5eeabdd79e5a0eeb8203922675e8e8503465b07c)